src/views/templates/utils/constants.ts

```diff
@@ -30,10 +30,8 @@ export type SOURCE_OPTIONS_IDS =
 
 export const SUPPORT_URL = 'https://access.redhat.com/articles/4234591';
 
-// t('You do not have permission to edit. To make changes to this template, contact your administrator.')
-export const NO_EDIT_TEMPLATE_PERMISSIONS =
-  'You do not have permission to edit. To make changes to this template, contact your administrator.';
+// t('Can not edit in view-only mode')
+export const NO_EDIT_TEMPLATE_PERMISSIONS = 'Can not edit in view-only mode';
 
-// t('You do not have permission to delete. To make changes to this template, contact your administrator.')
-export const NO_DELETE_TEMPLATE_PERMISSIONS =
-  'You do not have permission to delete. To make changes to this template, contact your administrator.';
+// t('Can not delete in view-only mode')
+export const NO_DELETE_TEMPLATE_PERMISSIONS = 'Can not delete in view-only mode';
```

Comment:
it doesn't seem these constants are being used, if not can they be used, or if not needed then removed?